model/src/commonMain/kotlin/orchestrator/OrchestratorMessage.kt

```diff
@@ -74,6 +74,16 @@ data class AnalyzerWorkerError(
     override val jobId: Long
 ) : OrchestratorMessage(), WorkerMessage
 
+/**
+ * A message notifying the Orchestrator about a result produced by the Analyzer Worker. This result has issues over the
+ * threshold.
+ */
+@Serializable
+data class AnalyzerWorkerResultWithIssues(
+    /** The ID of the Analyzer job, as it is stored in the database. */
+    override val jobId: Long
+) : OrchestratorMessage(), WorkerMessage
```

Comment:
I did the change.

For the future, I wonder if we could change the `OrchestratorMessage` with e.g.:

```
data class OrchestratorMessage(
    val jobId: Long,
    /** Analyzer, Scanner, ... */
    val type: OcherstratorMessageType
    /** Finished, FinishedWithErrors, ...*/
    val result: Status
)
```

and no subclass, to simplify even further the messaging and the amount of boilerplate code needed to be written.

Reply:
For structures like that sealed classes are often convenient, because when you handle them with a `when` block the compiler can already guarantee that all cases are handled. Actually, that would be an argument against my proposal.
But as we discussed we offline, we will have to look into this topic anyway outside of this PR.